rfcs/text/0008-threat-intel.md

```diff
@@ -291,6 +310,8 @@ Stage 1: Identify potential concerns, implementation challenges, or complexity.
 -->
 
 1. How to best represent malware{name,family,type}. Current proposal is to use `threat.indicator.classification` to describe threat delivery (Hacktool etc.) and family name.
+  - This can be represented through the use of the [`threat.software.*`](https://www.elastic.co/guide/en/ecs/master/ecs-threat.html) fields.
+  - Awaiting [approval](https://github.com/elastic/ecs/pull/1480#issuecomment-889312434) of this recommendation.
 1. Field types (Ref: https://github.com/elastic/ecs/pull/1127#issuecomment-776126293)
   - `threatintel.indicator.*` (Filebeat module) will be normal field type and will be deprecated when nested field types are better supported in Kibana
   - `threat.indicator.*` (actual threat ECS fieldset) will be nested now and used for enriched doc
```

Comment:
@peasead I don't think this is still true; the plan for `threat.indicator` is to keep it as a normal field, as is already the case with `threatintel.indicator`. RE the migration below, I think the only change on the data shipper side is to  rename `threatintel.indicator` to `threat.indicator`.

`threat.enrichments` is the nested field that will cover the enrichment use case; the caveats about kibana support still apply there for Detection Alerts and users manually adopting the `threat.enrichments` fields, but it should not impact data shippers at this time.

Reply:
Thanks, @rylnd so this isn't a "concern" any longer?

The concern may actually be that `threatintel.indicator.*` for the Threat Intel Filebeat module needs to be changed to `threat.indicator.*` once this is out of `[beta]`?

Reply:
Roger.

I'll update the Concern.

@ebeahan FYI ☝️